includes/wc-user-functions.php

```diff
@@ -108,7 +108,7 @@ function wc_create_new_customer( $email, $username = '', $password = '' ) {
 		$customer_id = wp_insert_user( $new_customer_data );
 
 		if ( is_wp_error( $customer_id ) ) {
-			return new WP_Error( 'registration-error', '<strong>' . __( 'ERROR', 'woocommerce' ) . '</strong>: ' . __( 'Couldn&#8217;t register you&hellip; please contact us if you continue to have problems.', 'woocommerce' ) );
+			return new WP_Error( 'registration-error', '<strong>' . __( 'Error', 'woocommerce' ) . ':</strong> ' . __( 'Couldn&#8217;t register you&hellip; please contact us if you continue to have problems.', 'woocommerce' ) );
```

Comment:
In other error messages the `:` was outside the translation strings. But you are right, we should move it inside the.

I'm planing to review all the error messages in a separate pull request.